packages/metadata/README.md

```diff
@@ -0,0 +1,35 @@
+# @loopback/metadata
+
+This module contains utilities to manipulate TypeScript metadata. It includes:
+
+* Reflector: wrapper of
+  [reflect-metadata](https://github.com/rbuckton/reflect-metadata)
+* Decorator factories: A set of factories for class/method/property/parameter
+  [decorators](https://www.typescriptlang.org/docs/handbook/decorators.html) to
+  apply metadata to a given class and its members.
```

Comment:
Could you please beef up this README? I'd like to make this module useful to people outside LoopBack community too, so that we can gain more usage and more contributions.  In order to get that, we need the README to be compelling enough for people who discover this package via search on npmjs.com.

See our README guidelines here: https://loopback.io/doc/en/contrib/README-guidelines.html. "Basic use" is the part I am missing most, e.g. how to use a decorator factory to create a decorator and then how to read the metadata stored by the decorator using the inspector.

I am ok with landing this pull request with the current insufficient README as long as the README is updated very soon afterwards, preferably before releasing this new package to npmjs.